content/en/docs/Installation/trustedboot.md

```diff
@@ -64,24 +59,63 @@ Substitute `$MY_ORG` for your own string, this can be anything but it help ident
 It is very important to preserve the keys generated in this process in a safe place. Loosing the keys will prevent you to generate new images that can be used for upgrades.
 {{% /alert %}}
 
+### Custom certificates by skipping Microsoft certificate keys
+
+{{% alert title="Warning" %}}
+Some firmware is signed and verified with Microsoft's or other vendor keys when secure boot is enabled. Removing those keys could brick them. It is preferable to not use the Microsoft certificates for precaution and security reasons, however as this might be potentially a dangerous action, only do this if you know what you are doing. To check if your firmware is signed with Microsoft's keys, you can check https://github.com/Foxboron/sbctl/wiki/FAQ#option-rom. See also: https://wiki.archlinux.org/title/Unified_Extensible_Firmware_Interface/Secure_Boot#Enrolling_Option_ROM_digests.
+{{% /alert %}}
+
+If your hardware supports booting with custom Secure Boot keys, you can optionally create keys from scratch and skip the Microsoft certificate keys. To do so, run the following command:
+
+```bash
+MY_ORG="Acme Corp"
+# Generate the keys
+docker run -v $PWD/keys:/work/keys -ti --rm quay.io/kairos/osbuilder-tools:latest genkey "$MY_ORG" --skip-microsoft-certs-I-KNOW-WHAT-IM-DOING --expiration-in-days 365 -o /work/keys
+```
+
+### Exporting keys from BIOS/UEFI and using them
+
+This is useful if you want to handle mass-installations where you don't want to enroll the generated keys manually in your bios/uefi.
+
+Some hardware might require additional vendor keys, that could be e.g. used to sign additional firmware. In order to re-use the same set of keys in the machine, you can export the keys from the BIOS/UEFI and use them to generate the UKI files.
+
+Some BIOS/UEFI allows to export the keys to USB stick directly from the BIOS/UEFI setup menu (for example [HPE](https://techlibrary.hpe.com/docs/iss/proliant-gen10-uefi/GUID-26E9F167-8061-4D85-9A0A-B610D00DFA3B.html)). If you have the keys in a USB stick, you can copy in a directory and use them to generate the UKI files.
+
+```bash
+MY_ORG="Acme Corp"
+MACHINE_CERTS="$PWD/path/to/machine-certs"
+# ~$ tree $PWD/path/to/machine-certs
+# $PWD/path/to/machine-certs
+# └── custom
+#     ├── db
+#     │   └── db
+#     ├── dbx
+#     ├── KEK
+#     │   └── KEK
+#     └── PK
```

Comment:
This is the directory structure in which the sbctl library expects them. `enki` accepts a directory structure of:

```
~/workspace/kairos/enki (import-custom-certs)*$ tree asus-exported-keys/
asus-exported-keys/
├── db
├── dbx
├── KEK
└── PK
```

and exports `der` versions in a directory structure like the one you have (in a temporary directory).
I did it like this because the directory strucure for the user is simpler and it was the one created by the Asus BIOS when I exported the keys.

See here: https://github.com/kairos-io/enki/pull/86/files#diff-b3cc45311045954f942496f625c5bd8fea1645ec4456a5ea3d123ed20e1dc33c